tools/cockpit.spec.in

```diff
@@ -613,9 +613,9 @@ Dummy package from building optional packages only; never install or publish me.
 %package -n cockpit-storaged
 Summary: Cockpit user interface for storage, using udisks
 Requires: cockpit-shell >= @REQUIRED_BASE@
-Requires: udisks2 >= 2.6
-Recommends: udisks2-lvm2 >= 2.6
-Recommends: udisks2-iscsi >= 2.6
+Requires: udisks2 >= 2.9
```

Comment:
Please make the equivalent change also for Debian and (possibly) Arch.

Reply:
Done for Debian, I ask how to do that for Arch. @jelly?

Reply:
Arch has only a single version of udisks2, which is 2.9.4 now and will never be lower, and they don't generally specify versioned dependencies, so I we don't need to either.

Reply:
Right.  That's what I was thinking.  I don't know how Arch-world works :)